test/unit/test_sdb.c

```diff
@@ -89,14 +85,17 @@ bool test_sdb_delete_none(void) {
 
 bool test_sdb_delete_alot(void) {
 	Sdb *db = sdb_new (NULL, NULL, false);
+	char buffer[256];
```

Comment:
```suggestion
	char buffer[32];
```

"key.%d" can be 4 + 11 + 1 as maximum, so 16 should be fine, but maybe better to give few more just to be safe in case of future changes